cvat-core/src/session.js

```diff
@@ -1559,6 +1560,21 @@
                          */
                         get: () => data.dimension,
                     },
+                    /**
+                     * @name cloudStorageId
+                     * @type {integer|null}
+                     * @memberof module:API.cvat.classes.Task
+                     * @instance
+                     */
+                    cloudStorageId: {
+                        get: () => data.cloud_storage_id,
+                        set: (cloudStorageId) => {
+                            if (!Number.isInteger(cloudStorageId) || cloudStorageId <= 0) {
+                                throw new ArgumentError('Value must be a positive integer');
+                            }
+                            data.cloud_storage_id = cloudStorageId;
+                        },
```

Comment:
Since we can't change cloudStorageId, I would suggest to do this field readonly (cloud_storage_id is set via constructor now). What do you think?